test/res.vary.js

```diff
@@ -5,7 +5,7 @@ var request = require('supertest');
 
 describe('res.vary()', function(){
   describe('with no arguments', function(){
-    it('should not set Vary', function (done) {
+    it('should error missing field', function (done) {
```

Comment:
Can you add this as an additional test rather than altering the existing test? I think both the former test and the test you changed it to have merit and would like to have both.